src/coreComponents/physicsSolvers/fluidFlow/wells/WellControls.hpp

```diff
@@ -366,14 +364,23 @@ class WellControls : public dataRepository::Group
   /// Tuning coefficient for the initial well pressure
   real64 m_initialPressureCoefficient;
 
+  /// Rate sign. +1 for injector, -1 for producer
+  real64 m_rateSign;
+
   /// Total rate table
-  TableFunction * m_targetTotalRateTable;
+  TableFunction const * m_targetTotalRateTable;
 
   /// Phase rate table
-  TableFunction * m_targetPhaseRateTable;
+  TableFunction const * m_targetPhaseRateTable;
 
   /// BHP table
-  TableFunction * m_targetBHPTable;
+  TableFunction const * m_targetBHPTable;
+
+  /// Default BHP for rate controlled injectors
+  static constexpr real64 maximumBHP = 1.01325e8;
+
+  /// Default BHP for rate controlled producers
+  static constexpr real64 minimumBHP = 1.01325e5;
```

Comment:
@francoishamon These values are hardcoded here. Is there a more appropriate place/policy for constants especially since they're dimensional.

Reply:
Currently we don't have a good place or policy to put the constants in the code, so what you have done is good. But maybe this is a good occasion to start something cleaner in the `wells` folder and start extracting the constants in a separate file. 

So, we could create a file `WellConstants.hpp` in `src/coreComponents/physicsSolvers/fluidFlow/wells` and put the `maximumBHP` and `minimumBHP` there with inline code documentation as you already did. What do you think?

Also, we can bring up this topic either in an issue or at the developer meeting, because there are currently quite a few hard-coded values buried inside the code.